app/code/core/Mage/Adminhtml/Block/Catalog/Product/Helper/Form/Gallery.php

```diff
@@ -37,9 +37,8 @@ class Mage_Adminhtml_Block_Catalog_Product_Helper_Form_Gallery extends Varien_Da
 
     public function getElementHtml()
     {
-        $html = $this->getContentHtml();
         //$html.= $this->getAfterElementHtml();
```

Comment:
can I suggest to replace the comment by `//return $this->getContentHtml().$this->getAfterElementHtml();`

Reply:
Separate PR if not already fixed.